resources/content/collections/categories.yaml

```diff
@@ -0,0 +1,13 @@
+title: Categories
+propagate: false
+template: default
+layout: layout
+revisions: false
+slugs: false
+sort_dir: asc
+date_behavior:
+  past: public
+  future: private
+structure:
+  root: false
+  slugs: true
```

Comment:
above there's `slugs: false`, what's the diference?

Reply:
This option shows the slug in the tree view in the CP. This was not necessary for the default view.